### ISSUE TITLE
User Signed In

### ISSUE THREAD

**Issue by jeffkush-sdi** [searchdiscovery/Apollo-Documentation-Connected-Vehicle#18]
[User Signed In](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/blob/1f284b5b3e98be9636ce29d275b85c2c736aacad/Data%20Layer%20Events/User%20Signed%20In.md)

**QA Jan 26:** [Comment](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/issues/18#issuecomment-1912789201)

**Comment by jeffkush-sdi** [comment]
@Shocopop
Hey Anton, so was trying to trigger this event during my QA and was unable to find an applicable link/page(?). Can you send me quick steps or URI that can facilitate QA on this event? Essentially this request applies to 'all' user sign-in related events.  thx in advance

**Comment by Shocopop** [comment]
@jeffkush-sdi This is fixed, user sign-in is now being fired on otp validation

**Comment by jeffkush-sdi** [comment]
@Shocopop So maybe I need to start from an alternate URI...IDK, but how I start is from here https://cwp.sit.siriusxmguardian.com/manage?u=64c17925-1ca4-4593-a2a5-261d6d975b08&v=bef0957f-673b-48bf-b1d6-53ae157ff65a&lang=en-US  > YOP get code > Enter code SXM

I dont see a sign in link anywhere(?) but I think someone on team mentioned [FWW] may not have a user yet(?) Not entirely sure of context here, but I vaguely recall, Is there a alternate URL you can pass along...I know you coded something. TY



![Image](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/assets/116372762/1d701e2a-4f79-4383-a333-343b0005fae4)

**Comment by Shocopop** [comment]
@jeffkush-sdi This event now fires after the OTP step

**Comment by jeffkush-sdi** [comment]
@Shocopop 

**QA .SIT**

**ISSUE** (minor) Now seeing this event fire; TY. The minor issue I see is the event schema is missing the `hashedEmail` parameter. This may not be possible, or entirely necessary. If we can do it...I would do it; but will review with @msawlorSDI  and confirm the need.

![Image](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/assets/116372762/cfd87d0a-0624-4035-8f97-a670811601e5)

![Image](https://github.com/searchdiscovery/Apollo-Documentation-Connected-Vehicle/assets/116372762/3d77a7b3-7ea0-4ae1-9e42-59808a9e88cf)

**Comment by Shocopop** [comment]
@jeffkush-sdi Yep, you are right, `hashedEmail` was missing in the schema that I have created in the code (for this event), thus it's not present, the change is very simple to do, two lines of code. I will make the PR, it will be merged either on Monday or tomorrow (if someone can approve my PR tomorrow). Although that change wouldn't be in staging for a while now (but it shouldn't be a blocking issue I guess)

**Comment by jeffkush-sdi** [comment]
@Shocopop see hashed email now,TY
